sst/test/src/tg1_load_store.cc

```diff
@@ -0,0 +1,192 @@
+#include "sstsimengtest.hh"
+
+TEST_GROUP(TG1, "SSTSimEng_correctly_handles_load_and_store_instructions",
+           "fastL1WithParams_config.py", "src", R"( mov x1, #1 )");
+
+TEST_CASE(TG1, "load_of_different_size_from_memory_64bits", "src", R"(
+    # Get heap address
+    mov x0, 0
+    mov x8, 214
+    svc #0
+    ldr x1, [x0]
+    )",
+          "348709988") {
```

Comment:
For all of these tests, where are the heap values being initialised?

Reply:
The values get initialised in the final process image starting from the heap address before being sent to SST, so they won't be in the cache.

Reply:
Sorry, I meant outside of the test. How / where is `348709988` being put into the heap? Where is this process image made?

Reply:
I just realised I was mis-understanding what was being done yesterday.......... This is clear now, thanks for the explanation!